tests/unit/EnvironmentTest.php

```diff
@@ -91,7 +91,7 @@ public function testSetConfig()
 
     public function testSetConfigAfterInit()
     {
-        $this->setExpectedException('RuntimeException');
+        $this->expectException('RuntimeException');
```

Comment:
As we need to keep `PHPUnit 4` compatibility, `expectException` shouldn't be used. Use `@expectedException` annotation instead.